pkgs/development/libraries/libgpod/default.nix

```diff
@@ -23,7 +25,7 @@ in stdenv.mkDerivation rec {
   dontStrip = true;
 
   propagatedBuildInputs = [ glib libxml2 sqlite zlib sg3_utils
-    gdk-pixbuf taglib libimobiledevice python pygobject2 mutagen ];
+    gdk-pixbuf taglib libimobiledevice  mutagen ];
```

Comment:
I feel like a lot of these shouldn't be propagated at all. Along with `dontStrip = true;` it inflates the closure size by a lot:

```
$ nix path-info -Sh ./result
/nix/store/1g3ipk8jqc7brx7bivhi52fqylw9v1fd-libgpod-0.8.3        417.6M
$ nix path-info -Sh ./result
/nix/store/5ha9yd4g7wik1qq0bawb57cb772k4ynp-libgpod-0.8.3         78.0M
```

Reply:
A lot of which are in `Requires`, so they have to be propagated for pkg-config to work correct.
But indeed it's overlinking, and there's a patch for it https://src.fedoraproject.org/rpms/libgpod/blob/master/f/libgpod-0.8.2-pkgconfig_overlinking.patch. But given I don't maintain this package, it doesn't get this kind of improvement in the scope of the PR :smile:

Reply:
only 4 of them are
```
Requires: glib-2.0 >= 2.8.0 gobject-2.0 gdk-pixbuf-2.0 libimobiledevice-1.0
```

Reply:
but I understand not want to shine an expression to a mirror-like shine when you're not very invested in it. I'm with with the current changes.